test/functional/operation_example.test.js

```diff
@@ -4105,7 +4105,7 @@ describe('Operation Examples', function() {
         // Create a capped collection with a maximum of 1000 documents
         db.createCollection(
           'a_simple_collection',
-          { capped: true, size: 10000, max: 1000, w: 1 },
+          { capped: true, size: 1000, max: 1000, w: 1 },
```

Comment:
What motivated this change?

Reply:
This was a mistake, I "fixed" the wrong test. I meant to fix [this line](https://github.com/mongodb/node-mongodb-native/blob/3.6/test/functional/operation_example.test.js#L9382) which sets a max of 10000. That looks like a typo to me based on the comment and the rest of the test.